CMakeLists.txt

```diff
@@ -25,8 +25,10 @@ set(PROJECT_VERSION_MINOR 4)
 set(PROJECT_VERSION_PATCH 0)
 option(SKIP_TESTS "Skips building all tests." OFF)
 option(PORTABLE "Instructs the compiler to remove architecture specific optimizations" ON)
+option(USE_SYSTEM_OPENSSL "Instructs the build system to search for and use an SSL library available in the host system" OFF)
```

Comment:
I'm not in favor of changing the system dependency in a minor release. I would be in favor of moving to USE_SYSTEM_OPENSSL as the default until the next major release.

Reply:
OK I'll invert that default. The thinking is that building/static-linking makes the binary more portable and that's probably the config we want for the release binary. But I agree it's a fairly significant change for a minor release.